src/content/docs/en/reference/api-reference.mdx

```diff
@@ -1703,6 +1703,27 @@ import { Code } from 'astro:components';
 
 This component provides syntax highlighting for code blocks at build time (no client-side JavaScript included). The component is powered internally by Shiki and it supports all popular [themes](https://shiki.style/themes) and [languages](https://shiki.style/languages). Plus, you can add your custom themes and languages by passing them to `theme` and `lang` respectively.
 
+#### Transformers
+
+<p><Since v="4.11.0" /></p>
+
+[Shiki transformers](https://shiki.style/packages/transformers#shikijs-transformers) can optionally be applied to code by passing them in through the `transformers` property as an array:
+
```

Comment:
```suggestion

Note that `transformers` only applies classes and you must provide your own CSS rules to target the elements of your code block.

```
This is the same suggestion I left in [withastro/astro/pull/11197](https://github.com/withastro/astro/pull/11197#discussion_r1644199714) -- something about needing to add your own CSS rules for styling.